guiEditor/src/diagram/workbench.tsx

```diff
@@ -203,151 +217,148 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
     onDown(evt: React.PointerEvent<HTMLElement>) {
         this._rootContainer.setPointerCapture(evt.pointerId);
 
-        if(!this.isOverGUINode) {
+        if (!this.isOverGUINode) {
             this.props.globalState.onSelectionChangedObservable.notifyObservers(null);
         }
 
         var pos = this.getGroundPosition();
-        this._mouseStartPointX = pos? pos.x : this._mouseStartPointX;
-        this._mouseStartPointY = pos? -pos.z : this._mouseStartPointY; 
-             
+        this._mouseStartPointX = pos ? pos.x : this._mouseStartPointX;
+        this._mouseStartPointY = pos ? -pos.z : this._mouseStartPointY;
     }
 
-    public isUp : boolean = true;
+    public isUp: boolean = true;
     onUp(evt: React.PointerEvent) {
         this._mouseStartPointX = null;
         this._mouseStartPointY = null;
-        this._rootContainer.releasePointerCapture(evt.pointerId);   
+        this._rootContainer.releasePointerCapture(evt.pointerId);
         this.isUp = true;
     }
 
     public createGUICanvas() {
-        
         // Get the canvas element from the DOM.
         const canvas = document.getElementById("workbench-canvas") as HTMLCanvasElement;
 
         // Associate a Babylon Engine to it.
         const engine = new Engine(canvas);
-        
+
         // Create our first scene.
         this._scene = new Scene(engine);
         this._scene.clearColor = new Color4(0.2, 0.2, 0.3, 1.0);
-        let camera = new ArcRotateCamera(
-            "Camera", -Math.PI / 2, 0, 1024, Vector3.Zero(), this._scene);
-        const light = new HemisphericLight(
-            "light1", Axis.Y, this._scene);
+        let camera = new ArcRotateCamera("Camera", -Math.PI / 2, 0, 1024, Vector3.Zero(), this._scene);

```

Comment:
`let` suggests that the camera might be reassigned later in the code. Is that the case here?